File: game-server/src/main/java/work/fking/pangya/game/packet/handler/MyRoomOpenedPacketHandler.java
package work.fking.pangya.game.packet.handler;

import io.netty.channel.Channel;
import work.fking.pangya.game.packet.inbound.MyRoomOpenPacket;
import work.fking.pangya.game.packet.inbound.MyRoomOpenedPacket;
import work.fking.pangya.networking.protocol.InboundPacketHandler;

public class MyRoomOpenedPacketHandler implements InboundPacketHandler<MyRoomOpenedPacket> {

    @Override
    public void handle(Channel channel, MyRoomOpenedPacket packet) {

    }
}

File: game-server/src/main/kotlin/work/fking/pangya/game/packet/handler/MyRoomOpenedPacketHandler.kt
package work.fking.pangya.game.packet.handler

import io.netty.buffer.ByteBuf
import work.fking.pangya.game.GameServer
import work.fking.pangya.game.net.ClientPacketHandler
import work.fking.pangya.game.player.Player

class MyRoomOpenedPacketHandler : ClientPacketHandler {
    override fun handle(server: GameServer, player: Player, packet: ByteBuf) {
        val selfPlayerUid = packet.readIntLE()
        val unknown = packet.readByte()
    }
}
